mix.exs

```diff
@@ -23,7 +23,7 @@ defmodule PhoenixSlime.Mixfile do
     [{:phoenix, "~> 1.3-rc"},
      {:phoenix_html, "~> 2.6"},
      {:cowboy, "~> 1.0"},
-     {:slime, "~> 0.16"},
+     {:slime, "~> 1.0"},
```

Comment:
Would 1.0.0 be preferred over 1.0?

~> 1.0.0	>= 1.0.0 and < 1.1.0
~> 1.0	>= 1.0.0 and < 2.0.0